app/models/aws_s3.rb

```diff
@@ -87,7 +87,7 @@ def fetch_key_list(prefix: '')
     @bucket.objects(prefix: prefix).sort_by(&:last_modified).map(&:key)
   end
 
-  def list_objects(max_keys = 1_000, prefix: '')
+  def list_objects(max_keys = 10_000, prefix: '')
```

Comment:
I don't think we should change the default for everyone, that might have some wide-ranging impacts that we are unaware of. We can pass the `max_keys` argument to list_objects from ConsumeExternalFormSubmissionsJob instead.

Reply:
Oh, thanks good catch. It looks like the s3 api won't allow a max keys of 10,000 anyway.

Reply:
ahh k good call to revert